File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
       int cnt = 1;
       int i = 0, j = 1;
        while(j<nums.size())
        {
            while(j<nums.size() && nums[i]==nums[j])
                j++;
            if(j<nums.size() && nums[i]!=nums[j])
            {
                nums[++i] = nums[j++];
                cnt++;
            }
        }  
        return cnt; 
    }
};

auto init = []
{
    ios::sync_with_stdio(NULL);
    cin.tie(NULL);
    cout.tie(NULL);
    return 'c';
}();

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i = 0
        j = 0
        while(j<len(nums)):
            while(j < len(nums) and nums[j] == nums[i]):
                j+=1
            if(j < len(nums)):
                nums[i+1] = nums[j]
                i+=1
        return i+1
        
